### PR TITLE
French translation for Incus

### PR DESCRIPTION
**Pull request by bensmrs**
This PR translates the /incus section of the website into French. Note that the `static/js/tryit.js` file has been modified not to rely on English button values anymore, but further work is needed to localize other website elements (see i.e. #725)

### CODE REVIEW

File: static/js/tryit.js [hunk 1]
```diff
@@ -457,7 +457,7 @@ $(document).ready(function() {
         var displayFeedback = 0;
```
Comment (stgraber) [inline static/js/tryit.js:457]: This change should be in a separate commit.

**Comment by stgraber** [comment]
Looks good, just that one change which isn't a french translation and so should be in its own commit.

**Comment by bensmrs** [comment]
You’re right, that should be good now!

**Comment by stgraber** [comment]
Looks good, thanks!
